compiler/src/iree/compiler/Codegen/Common/GPU/GPUApplyTilingLevel.cpp

```diff
@@ -135,6 +135,12 @@ applyTileAndFuseToEachRoot(RewriterBase &rewriter,
             bool isDestinationOperand)
         -> std::optional<scf::SCFTileAndFuseOptions::ControlFnResult> {
       Operation *owner = originalProducer.getOwner();
+      if (tilingLevel == IREE::GPU::TilingLevel::Reduction &&
```

Comment:
[optional] While you're here, I think we also want to turn it off for Subgroup level